modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowNotifications.testcase

```diff
@@ -112,12 +112,46 @@ definition {
 	}
 
 	@description = "Verify that the user can open new user details by notifications"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOpenNewUserDetailsByNotifications {
+		property test.name.skip.portal.instance = "WorkflowNotifications#CanOpenNewUserDetailsByNotifications";
 
-		// TODO LRQA-70881 CanOpenNewUserDetailsByNotifications pending implementation
+		task ("Give configure Workflow to Single Approver for User") {
+			Workflow.openWorkflowListView();
 
+			Navigator.gotoNavItem(navItem = "Configuration");
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "User");
+		}
+
+		task ("When create a New User") {
+			WorkflowUser.disableCaptchaToCreateNewUserAndSendPassword();
+
+			User.logoutPG();
+
+			WorkflowAsset.createNewUserAccount(
+				newPassword = "password",
+				userEmailAddress = "user@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+		}
+
+		task ("Then view a created user's notification and details with the admin account") {
+			User.loginUserPG(
+				password = "test",
+				userEmailAddress = "test@liferay.com");
+
+			WorkflowAsset.gotoViaNotification(assetType = "User");
+
+			WorkflowAsset.viewUserDetails(userEmailAddress = "user@liferay.com");
+
+			Workflow.gotoPreviewView();
+
+			WorkflowAsset.viewUserDetails(userEmailAddress = "user@liferay.com");
```

Comment:
I don't think these last two steps are necessary since you are already asserting the user email in the notification view. 

If you do need to include the Preview View portion then you can delete the first _WorkflowAsset.viewUserDetails_ instead.